announcement/views/announcement_views.xml

```diff
@@ -0,0 +1,239 @@
+            <list>
+                <field name="create_date" />
+                <field name="create_uid" />
+            </list>
+        </field>
+    </record>
+    <record id="action_announcement_log" model="ir.actions.act_window">
+        <field name="name">Announcement Logs</field>
+        <field name="res_model">announcement.log</field>
+        <field name="context">{}</field>
+        <field name="domain">[('announcement_id', '=', active_id)]</field>
+        <field name="view_mode">list</field>
+    </record>
+    <record id="announcement_view_tree" model="ir.ui.view">
+        <field name="model">announcement</field>
+        <field name="arch" type="xml">
+            <list>
+                <field name="name" />
+                <field name="notification_date" />
+                <field name="notification_expiry_date" />
+                <field
+                    name="tag_ids"
+                    optional="hide"
+                    widget="many2many_tags"
+                    options="{'color_field': 'color'}"
+                />
+            </list>
+        </field>
+    </record>
+    <record id="announcement_management_view_tree" model="ir.ui.view">
+        <field name="model">announcement</field>
+        <field name="inherit_id" ref="announcement_view_tree" />
+        <!-- <field
+            name="groups_id"
+            eval="[(4, ref('announcement.announcemenent_manager'))]"
+        /> -->
+        <field name="arch" type="xml">
+            <field name="name" position="before">
+                <field name="sequence" widget="handle" />
+            </field>
+            <field name="notification_expiry_date" position="after">
+                <field name="read_announcement_count" string="Reads" />
+                <field name="allowed_users_count" string="Total users" />
+            </field>
+        </field>
+    </record>
+    <record id="announcement_view_form" model="ir.ui.view">
+        <field name="model">announcement</field>
+        <field name="arch" type="xml">
+            <form string="Announcement">
+                <sheet>
+                    <div class="oe_button_box" name="button_box" />
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        invisible="active"
+                    />
+                    <div class="oe_title">
+                        <h1>
+                            <field
+                                name="name"
+                                required="1"
+                                placeholder="e.g. Announcement description..."
+                            />
+                        </h1>
+                    </div>
+                    <group>
+                        <field name="is_general_announcement" />
+                    </group>
+                    <group>
+                        <field name="active" invisible="1" />
```

Comment:
```suggestion
```
No need to add invisible fields anymore, odoo will auto-add them if needed, right?